src/ol_concourse/pipelines/infrastructure/concourse/instance_refresh.py

```diff
@@ -10,6 +11,10 @@
 environments = ["ci", "qa", "production"]
 node_classes = ["worker-infra", "worker-ocw", "worker-generic", "web"]
 
+build_schedule = schedule(
+    Identifier("build-schedule"), interval="24h", start="08:00", stop="11:00"
```

Comment:
Presumably this is in UTC? So between 3AM and 6AM daily?

Reply:
Yup that was Carey's request in the issue. That window is timed around critical OCW work that can't be interrupted.

Thanks for checking!